### ISSUE TITLE
[Improvement] Audio Records with Ear ListenMode

### ISSUE THREAD

**Issue by DoM1niC** [RocketChat/Rocket.Chat.ReactNative#1091]
Just if you handset the Phone to Ear to listen the Message only there like Phone Calls.

**Comment by diegolmello** [comment]
@DoM1niC Thanks for opening. It would be a really cool feature.

**Comment by Nemental** [comment]
@diegolmello i think this is a key feature for mobile messenger in 2020 😅

**Comment by diegolmello** [comment]
@Nemental we're going to implement it on next versions. 4.7 or 4.8.

**Comment by Nemental** [comment]
> @Nemental we're going to implement it on next versions. 4.7 or 4.8.

Yeah thank you, great news 🤙🏼

**Comment by Klotz95** [comment]
Is currently someone working on this one? If not i would start to implement this feature. ￼

**Comment by diegolmello** [comment]
Not really. We had to change our plans back then (as everyone else this year).
I'd appreciate your contribution, @Klotz95.
Thanks!
